src/domains/home/components/LoginOverlay/ConnectOverlay.tsx

```diff
@@ -21,27 +99,91 @@ export const ConnectOverlay = () => {
 
         <div>
           <p className="text-md mb-4 text-theme-gray-500 dark:text-theme-gray-300">
-            {t("CONNECT_ARK_CONNECT_TO_START")}
+            {installationStatus === InstallationStatus.NONE
+              ? t("INSTALL_EXTENSION")
+              : t("CONNECT_ARK_CONNECT_TO_START")}
           </p>
 
           <div className="w-2/3 mx-auto">
-            {darkMode ? <ConnectDark /> : <ConnectLight />}
+            <Icon />
           </div>
         </div>
 
-        <div className="h-12 mt-6 flex flex-col items-center justify-center">
-          {!isConnecting && (
-            <Button
-              onClick={() => {
-                void connect();
-              }}
-            >
-              {isErrored && t("RETRY")}
-              {!isErrored && t("CONNECT")}
-            </Button>
+        <div
+          className={cn(
+            "h-12 mt-6 flex flex-col items-center justify-center min-h-fit w-full",
+            {
+              "flex-col-reverse":
+                installationStatus === InstallationStatus.METAMASK,
+            },
+          )}
+        >
+          {!connecting && (
+            <>
+              {!arkInstalled && (
+                <LinkButton
+                  href={arkExtensionUrl}
+                  isExternal
+                  variant="transparent"
+                  className="mt-2 w-full border rounded-lg border-[#d3d3d3] dark:border-theme-gray-500 justify-start px-5 py-4 group hover:border-transparent dark:hover:border-transparent"
```

Comment:
can we use `theme-gray-200` for these borders (and other instances where `#d3d3d3` is used) so they match with the color palette? that's also what the table uses in light mode for its borders